microcosm_flask/conventions/discovery.py

```diff
@@ -0,0 +1,60 @@
+"""
+A discovery endpoint provides links to other endpoints.
+
+"""
+from flask import jsonify
+
+from microcosm.api import defaults
+from microcosm_flask.linking import Link, Links
+from microcosm_flask.naming import name_for, singleton_path_for
+from microcosm_flask.paging import Page
+from microcosm_flask.operations import Operation
+
+
+def iter_search_operations(graph, operations):
+    for rule in graph.flask.url_map.iter_rules():
+        if "." in rule.endpoint:
+            operation, obj = Operation.parse(rule.endpoint)
+            if operation in operations:
+                yield operation, obj
+
+
+@defaults(
+    name="all",
```

Comment:
The default behavior will be to serve a discovery endpoint at `/all` for all "search" endpoints. This can obviously be tuned. I agonized a bit over `/all` or just `/`